safe_transaction_service/history/services/transaction_service.py

```diff
@@ -210,7 +210,7 @@ def get_all_tx_identifiers(
         module_tx_ids = (
             ModuleTransaction.objects.filter(safe=safe_address)
             .annotate(
-                execution_date=F("internal_tx__timestamp"),
+                execution_date=F("created"),
```

Comment:
`created` field is not the same than the block timestamp, it is a auto created field and would be the date when the transaction was indexed. 
Could we have issues because there are inserted transactions out of order? I mean recovered transactions because a reindex.

Reply:
It's the same due to our implementation: https://github.com/safe-global/safe-transaction-service/blob/c9f762bbc53a5afd4895992e059b6034d5559916/safe_transaction_service/history/indexers/tx_processor.py#L535

Reply:
Ok I see but is a little bit confusing that we are using AutoCreatedField. https://github.com/safe-global/safe-transaction-service/blob/4631ecd691e32b4b1cf1309c44b0eabd4a8a1e27/safe_transaction_service/history/models.py#L1512
Maybe we should thinking about add the created field to the model and remove this dependency?

Reply:
I agree and that makes sense, but we are doing the same for other models, so right now I would leave it that way